features/step_definitions/machine_set.rb

```diff
@@ -1,7 +1,8 @@
 Given(/^I pick a random machineset to scale$/) do
   ensure_admin_tagged
+  # MachineSets with replicas=0 or have taints are intentionally not picked
   machine_sets = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api")).
-    select { |ms| ms.available_replicas >= 1 }
+    select { |ms| ms.available_replicas >= 1 && ms.taints.nil? }
```

Comment:
Isn't overwriting them better than trying to find MS without taints? Leaving this as only a question for future consideration. I don't mind this as long as it is working for you.

Reply:
I  mean LGTM on my end.

Reply:
Yes, I could overwrite it, thank you.